### PR TITLE
Capella mainnet release

### PR DESCRIPTION
**Pull request by djrtwo**
todo:
* [x] bump VERSION.txt (#3326)
* [x] testvector gen

### CODE REVIEW

**Comment by mcdee** [comment]
Any chance of having a final release for Capella now that we've gone live on mainnet?  The beacon APIs are waiting on a release version so they can in turn release.

**Review by hwwhww — APPROVED** [review]
Test vectors are ready! 

To other repo maintainers: please don't merge other PRs that change test vectors. 🙏

**Comment by hwwhww** [comment]
@mcdee Sorry for the delays! It should be published in the next few hours.

**Pull request merged**
